src/topology/algebra/module.lean

```diff
@@ -283,6 +283,7 @@ end
 
 instance : has_one (M →L[R] M) := ⟨id R M⟩
 
+lemma one_def : (1 : M →L[R] M) = id R M := rfl
```

Comment:
`@[simp]`?  (And the next one?)

Reply:
The next one follows from `coe_id'`. I don't think that we should simplify `1` to `id`: what if someone wants to deal with the semiring of endomorphisms as a semiring?